content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+=============
+
+After verifying the changes, move the |ECO| to a :ref:`verification stage <plm/eco/stage-config>`.
+Once the approvers accept the changes, the :guilabel:`Apply Changes` button becomes available. Click
+this button, and the |ECO| is automatically closed, and the changes are applied.
+
+To ensure the changes are live, navigate to the :guilabel:`Revision` smart button.
+
+The |BOM| will have the large red :guilabel:`Archived` tag removed. Additionally, navigate to the
+product form, switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field will be
+updated to match the version number shown on the :guilabel:`Revision` smart button of the latest
+|ECO|.
+
+.. example::
+   After applying the changes of the |ECO| for the :ref:`keyboard <plm/eco/example-keyboard>`, view
+   the version of the current keyboard |BOM| in the :guilabel:`Miscellaneous` tab. Here, the
+   :guilabel:`Version` number has been updated to `2`, matching the `V2` that appears in the
+   :guilabel:`Revision` smart button of the |ECO|.
+
+   .. image:: engineering_change_orders/BOM-version.png
+      :align: center
+      :alt: View current *BOM* version in the Miscellaneous tab.
+
+.. _plm/eco/tablet-view:
+
+Create ECO from tablet view
+===========================
+
+|ECOs| created in *tablet view* are intended for operators to directly suggest clearer operation
+instructions. To create an |ECO|, navigate to the :guilabel:`Work Orders` tab on an ongoing
+manufacturing order. Then, click the :guilabel:`tablet icon` for the desired work order to open the
+tablet view of the operation.
+
+.. image:: engineering_change_orders/tablet-icon.png
+   :align: center
+   :alt: Find the tablet icon for each operation, second from the far right.
+
+Next, add an instructional step, by clicking the :guilabel:`☰ (three horizontal lines)` icon in the
+tablet view of an operation. Then, click the :guilabel:`Add a step` button.
+
+.. image:: engineering_change_orders/additional-options-menu.png
+   :align: center
+   :alt: Navigate to the "Add a Step" pop-up by clicking the three horizontal lines icon in tablet
+         view.
+
+In the :guilabel:`Title` field, enter a short step description. Next, in the
+:guilabel:`Instructions` text field, type the instructions of the step in greater detail.
+Optionally, add an image to the :guilabel:`Document` field. Once completed, finish by clicking the
+:guilabel:`Propose Change` button.
+
+.. example::
+   To propose an additional check for broken components, enter the details in the :guilabel:`Add a
+   Step` pop-up window. Doing so creates an instructional quality control point that will be
+   reviewed in the following section.
+
+   .. image:: engineering_change_orders/add-a-step.png
+      :align: center
+      :alt: Fill out the *Add a Step* form to suggest an additional quality control point.
+
+View ECO
+--------
+
+To review the proposed changes, go to the `BOM Changes` ECO type from :menuselection:`PLM app -->
```

Comment:
I'm only seeing BOM Updates on the PLM Overview page...is that the same thing?

Reply:
Yup! The cards on the PLM Overview page are like sales teams or projects. Everyone's ECO types can be different. From what I've seen, our runbots have the New Product Introduction and BOM updates eco types created by default, but sometimes they're deleted...